### ISSUE TITLE
XML integry check inside the code

### ISSUE THREAD

**Issue by GoogleCodeExporter** [fsrrt/wikiteam#34]
```
now handy on the tutorial (last section) 
http://code.google.com/p/wikiteam/wiki/Tutorial


```

Original issue reported on code.google.com by `emi...@gmail.com` on 15 Jul 2011 at 8:52
- Blocking: #75

**Comment by GoogleCodeExporter** [comment]
```
Somehow fixed by the new launcher.py, but not closing the bug as AFAICS the 
result is printed but not acted upon.
```

Original comment by `nemow...@gmail.com` on 9 Apr 2012 at 8:23

**Comment by GoogleCodeExporter** [comment]
```
Fixed in r710. Please reopen this issue if the change breaks.
```

Original comment by `ad...@alphacorp.tk` on 22 Jun 2012 at 11:35
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Reopening issue. Emijrp pointed out that this check breaks Windows-runners.

Probably need to check whether the user is running Linux or Windows.
```

Original comment by `ad...@alphacorp.tk` on 8 Aug 2012 at 10:41
- Changed state: **New**

**Comment by GoogleCodeExporter** [comment]
Original comment by `ad...@alphacorp.tk` on 12 Nov 2013 at 6:38
- Now blocking: #75

**Comment by GoogleCodeExporter** [comment]
```
Really fixed in r877, already patched in DumpGenerator 2.0.
```

Original comment by `ad...@alphacorp.tk` on 16 Nov 2013 at 1:40
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Have you tested that with big wikis? I hope it doesn't load the XML in memory; 
we also need to check how long it is in python to perform 5 checks against 
millions rows.
```

Original comment by `nemow...@gmail.com` on 17 Nov 2013 at 11:17
